src/test/resources/payloads/k8s-svc/bundles/bundle_with_versions.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
Try to add to the test more complicated versions, and also with or without the v in front
What I mean is you can try to use also versions like v0.0.1-alpha, v0.0.1-beta, or other valid combination. So we can test things are actually sorted correctly

Reply:
In this way aren't we testing the external library?
However I'm not sure if you have already seen the tests in the java class, because there are some tests with and witouh
 the leading v, and also 1 or 2 with the alpha. I start parsing this json file and then I customize the versions.

Reply:
For sure I have to add tests to check that specifying a string version we obtain the expected java-semver Version

Reply:
I think this is not about testing the external library, but making sure it works the way we expect it to work.